src/styles/normalize.css

```diff
@@ -0,0 +1,9 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
```

Comment:
In code, you are using font-weight 400,500,700, but from google-fonts you download only 500, that's why you can't pass the test. Add 400 and 700 on the website and change this link.

Reply:
Thanks for your advice, it really helped.